docs/DevelopmentGuide.md

```diff
@@ -80,7 +80,7 @@ the default Visual Studio style. See [clang-format style options](
 http://releases.llvm.org/3.6.0/tools/clang/docs/ClangFormatStyleOptions.html)
 for details.
 
-If you see unexpected formatting changes in the code, verify that you are running version 10 or higher of the LLVM tool-chain.
+If you see unexpected formatting changes in the code, verify that you are running version 11 or higher of the LLVM tool-chain.
```

Comment:
if we are not supportng 12+, then we should reword this?

Reply:
Maybe.  ebpf will work, but things as complicated as the ebpf-for-window-demo programs will fail verification due to the optimizations done by LLVM 12+.